docs/setup/settings.asciidoc

```diff
@@ -456,6 +456,14 @@ identifies this {kib} instance. *Default: `"your-hostname"`*
 {kib} is served by a back end server. This
 setting specifies the port to use. *Default: `5601`*
 
+`server.protocol`::
```

Comment:
Can we add this to [kibana-docker](https://github.com/elastic/kibana/blob/5715ee8347b5b05393b16a860623bba090c545f7/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)?

Reply:
Woups, sorry - https://github.com/elastic/kibana/pull/183465/commits/ecebcbd82a1128aeea9cf30d956d07decc71457f